Pod/Classes/Acknowledgements/AcknowledgementViewController.swift

```diff
@@ -69,7 +69,7 @@ private extension AcknowledgementViewController {
         labelView.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             labelView.widthAnchor.constraint(equalTo: view.layoutMarginsGuide.widthAnchor),
-            labelView.topAnchor.constraint(equalTo: view.topAnchor),
+            labelView.topAnchor.constraint(equalTo: view.topAnchor, constant: 16.0),
```

Comment:
@ZevEisenberg: Not always the case. LicenseFormatter doesn't seem to consistently include a newline at the end of `FooterText`.